packages/class-name-updater/package.json

```
@@ -16,7 +16,8 @@
   "scripts": {
     "build": "tsc --build --verbose ./tsconfig.json",
     "clean": "rimraf ./dist",
-    "test": "node ./dist/cli.js test --exclude test/largeFile.js"
+    "test": "node ./dist/cli.js test --exclude test/largeFile.js",
+    "test:v6": "node ./dist/cli.js test --exclude test/largeFile.js --pfVersion 6"
```

Actually there's a cleaner way we can do this than what I had suggested before 😅 

```suggestion
    "test": "yarn build && yarn test:v5 && yarn test:v6",
    "test:v5": "node ./dist/cli.js test --exclude test/largeFile.js"
    "test:v6": "node ./dist/cli.js test --exclude test/largeFile.js --pfVersion 6"
```